src/Sulu/Bundle/SecurityBundle/Entity/BaseUser.php

```diff
@@ -156,6 +156,10 @@ public function getUsername()
      */
     public function getFullName()
     {
+        if (!$this instanceof User) {
```

Comment:
How does that make sense?

Reply:
call undefined method `getContact` on BaseUser is the error.
so method_exists or instanceof check is needed before. or what would you change?

Reply:
Why don't we implement this  method in the `User` class instead?

Reply:
seems like method also exist in `User` class 🙈 https://github.com/sulu/sulu/blob/e8600ca4e16275e67a2f9ff9432021c603a21c7c/src/Sulu/Bundle/SecurityBundle/Entity/User.php#L242